pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vmware_okms_vm-encrypt/guide.fr-fr.md

```diff
@@ -0,0 +1,990 @@
+
+#### 1. Ajouter un fournisseur de clé KMS
+
+Pour que vCenter puisse truster votre serveur KMS OVHcloud, nous avons besoin d'accéder à la console vSphere de votre PCC HPC VMware on OVHcloud.
+
+Pour accéder à l'url de connexion vSphere depuis un navigateur web (avec de preference une base navigateur chromium).
+
+Vous pouvez y accéder depuis le control panel OVHcloud, en cliquant sur:
+- `Hosted Private Cloud > VMware > Votre PCC > Manager interface > Web interface`{.action}.
+
+Une fois que vous avez cliqué sur Web interface, cliquez sur `vSphere HTML Client`{.action}.
+
+![Manager Web Interface](/pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vmware_okms_vm-encrypt/images/manager_web_interface_pcc.png){.thumbnail}
+
+Vous êtes maintenant sur l'url de votre pcc, par exemple: 
+- `<https://pcc-x.x.x.x.ovh.de/ui/>`{.action}
+
+Connectez-vous avec un utilisateur local ou avec un utilisateur IAM selon les droits que vous avez mis en place au sein de votre compte OVHcloud et de votre PCC HPC on OVHcloud.
+
+Vous êtes maintenant logué au sein de votre Hosted Private Cloud vSphere on OVHcloud.
+
+Pour accéder à la gestion des serveurs KMS dans vSphere, cliquer sur: `Configurer`{.action} depuis votre **pcc-XXX-XXX-XXX-XXX**.
+
+Puis, allez dans la section **Sécurité**: `Fournisseurs de clés`{.action}.
+
+Il vous faut maintenant, cliquez sur: `Ajouter > Ajouter un fournisseur de clé standard`{.action}.
+
+![KMS Key Provider](/pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vmware_okms_vm-encrypt/images/kms_key_provider.png){.thumbnail}
+
+Une fois que vous avez sélectionné l'option pour ajouter un Key Provider, une fenêtre ou un formulaire s'ouvrira pour saisir les détails du Key Provider que vous souhaitez ajouter. Cela peut inclure des informations telles que l'adresse IP ou le nom de domaine (DNS) du serveur Okms, ainsi que le port utilisé (5696).
+
+Les noms de domaines et le port (KMIP) ne changent pas.
+
+![KMS Key Provider](/pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vmware_okms_vm-encrypt/images/okms_vsphere_add_standard_key_provider.png){.thumbnail}
+
+Vous avez ces champs disponibles:
+
+| Champ       | Input                                                   | Description                                                                  |
+|-------------|---------------------------------------------------------|------------------------------------------------------------------------------|
+| **Name**    |                                                         | - Un champ qui permet de nommer votre cluster au sein de vCenter.            |
+| **KMS**     |                                                         | - Le nom qui apparaitra au sein de Vsphere pour votre Okms.                  |
+| **Address** | eu-west-rbx.okms.ovh.net <br/> eu-west-sbg.okms.ovh.net | - L'endpoint du serveur Okms, privilégier le nom de domaine plutôt que l'IP. |
```

Comment:
```suggestion
| **Address** | eu-west-rbx.okms.ovh.net <br/> eu-west-sbg.okms.ovh.net | **Endpoint** du serveur OKMS. Privilégiez le nom de domaine plutôt que l'IP. |
```

Reply:
Really ? because since the begining of the guide you told that it's preferable to use IP.